persistence/trees/trees.go

```diff
@@ -210,24 +213,30 @@ func (t *treeStore) updateMerkleTrees(pgtx pgx.Tx, txi indexer.TxIndexer, height
 		}
 	}
 
-	if err := t.commit(); err != nil {
+	if err := t.Commit(); err != nil {
 		return "", fmt.Errorf("failed to commit: %w", err)
 	}
 	return t.getStateHash(), nil
 }
 
-func (t *treeStore) commit() error {
-	for treeName, stateTree := range t.merkleTrees {
-		if err := stateTree.tree.Commit(); err != nil {
-			return fmt.Errorf("failed to commit %s: %w", treeName, err)
+func (t *TreeStore) Commit() error {
```

Comment:
Should we leave a comment in the case that one of the trees gets committed but another doesn't?

A race condition corruption error sort of thing?

Reply:
Like a log?

Reply:
Could be a log, or a TODO comment

Are we handling this in the "atomic rollbacks" PRs later?

Eg.
```
Commit Tree #1
Commit Tree #2
Node Crashes
Did not commit Tree #3
Did not commit Tree #4
```

Reply:
Logs added for now. This function could use a multierror and return a combined error for all of the commit calls it makes instead. I'll think about this a little bit more.